### PR TITLE
Fix multiple excel files for one config

### PR DESCRIPTION
**Pull request by pierce-martin**
Fix sending data when a single config is used for multiple excel files (power/energy are split across two files by sunpower).

### CODE REVIEW

File: sunpowerTochords.py [hunk 1]
```diff
@@ -91,29 +91,42 @@ def handleFile(config, file):
...
-                vars['at'] = int(df_row['Unix Timestamp'])
-                # Collect the vars
-                for (short_name, val) in zip (short_names,  [df_row[x] for x in column_names]):
-                    vars[short_name] = val
-                # Send to CHORDS
-                sendData(config=config, vars=vars)
-
-    
+
+        # Make sure period exists
```
Comment (pierce-martin) [inline sunpowerTochords.py:101]: Why is this needed?

**Review by sugartechllc — APPROVED** [review]
Very clean
